Makefile

```diff
@@ -80,7 +80,6 @@ go-mod:
 
 verify: verify-gofmt go-mod test
 
-pkg/api/generated/openapi/zz_generated.openapi.go:
-	GO111MODULE=off go get -d k8s.io/kube-openapi || true
+pkg/api/generated/openapi/zz_generated.openapi.go: go.mod go.sum
 	rm -rf pkg/api/generated/openapi
 	cd $(OPENAPI_PATH) && go run ./cmd/openapi-gen/openapi-gen.go --logtostderr -i k8s.io/metrics/pkg/apis/custom_metrics,k8s.io/metrics/pkg/apis/custom_metrics/v1beta1,k8s.io/metrics/pkg/apis/custom_metrics/v1beta2,k8s.io/metrics/pkg/apis/external_metrics,k8s.io/metrics/pkg/apis/external_metrics/v1beta1,k8s.io/metrics/pkg/apis/metrics,k8s.io/metrics/pkg/apis/metrics/v1beta1,k8s.io/apimachinery/pkg/apis/meta/v1,k8s.io/apimachinery/pkg/api/resource,k8s.io/apimachinery/pkg/version,k8s.io/api/core/v1 -p github.com/directxman12/k8s-prometheus-adapter/pkg/api/generated/openapi -O zz_generated.openapi -r /dev/null
```

Comment:
while we are at it, would you mind updating this command to make it gopath independent? Like we did here: https://github.com/kubernetes-sigs/custom-metrics-apiserver/pull/76/commits/97bc9e2bbd45c40a6de998965e713897099ef9ce

Reply:
Changed